azure-proto-core/SubscriptionOperations.cs

```diff
@@ -33,27 +32,29 @@ internal SubscriptionOperations(ArmClientContext context, Resource subscription,
         }
 
         public override ResourceType ResourceType => AzureResourceType;
-        internal SubscriptionsOperations SubscriptionsClient => GetClient<ResourcesManagementClient>((uri, cred) => 
+
+        internal SubscriptionsOperations SubscriptionsClient => GetClient<ResourcesManagementClient>((uri, cred) =>
             new ResourcesManagementClient(uri, Guid.NewGuid().ToString(), cred, ArmClientOptions.Convert<ResourcesManagementClientOptions>(ClientOptions))).Subscriptions;
-        internal ResourceGroupsOperations RgOperations => GetClient<ResourcesManagementClient>((uri, cred) => 
+
+        internal ResourceGroupsOperations RgOperations => GetClient<ResourcesManagementClient>((uri, cred) =>
             new ResourcesManagementClient(uri, Id.Subscription, cred, ArmClientOptions.Convert<ResourcesManagementClientOptions>(ClientOptions))).ResourceGroups;
 
         public Pageable<ResourceGroupOperations> ListResourceGroups(CancellationToken cancellationToken = default)
         {
-            return new PhWrappingPageable<ResourceGroup, ResourceGroupOperations>(
+            return new PhWrappingPageable<Azure.ResourceManager.Resources.Models.ResourceGroup, ResourceGroupOperations>(
```

Comment:
Why did we have to full specify this now? Azure.ResourceManager.Resources.Models.ResourceGroup from ResourceGroup.

Reply:
In order to differentiate between azure_proto_core.ResourceGroup and Azure.ResourceManager.Resources.Models.ResourceGroup